### ISSUE TITLE
Automation of projects visible in request form

### ISSUE THREAD

**Issue by Izhmash** [CCI-MOC/moc-openstack-tools#95]
It would be beneficial to automate the process of adding OpenStack projects to the visible drop-down list in the project request form.  This would require some exploration of the Google API and some further administrative decisions.  

One issue about this that was discussed during the meeting on 10/19 was about deciding which projects should be available.  For example, if the automation is setup to show all OpenStack projects, it presents the possibility of a user selecting and being approved for a project that they really shouldn't be in.

**Comment by greyspectrum** [comment]
Agreed that this would be an important feature. At the very least, projects like "admin" and "moc-infrastructure" shouldn't be included.
